tiledb/sm/group/group.cc

```diff
@@ -729,7 +752,49 @@ Group::member_by_index(uint64_t index) {
     uri = group_uri_.join_path(member->uri().to_string()).to_string();
   }
 
-  return {Status::Ok(), uri, member->type()};
+  return {Status::Ok(), uri, member->type(), member->name()};
+}
+
+tuple<
```

Comment:
Why are we returning a tuple here rather than `GroupMember`? Don't need to change it now, but IMO this should be done when we forward-port to 2.9

Reply:
No reason not to. I modified the URI for relative which is the only logic that needs to be accounted for in `Group::member_by_name` and `Group::member_by_index`. I'd like to open a story and do this in a followup if that is okay with you?

Reply:
👍